portfolio/src/main/java/com/google/sps/servlets/AddCommentsServlet.java

```diff
@@ -75,4 +79,27 @@ private String convertToJsonUsingGson(ArrayList<String> comments) {
     String json = gson.toJson(comments);
     return json;
   }
+
+  /** Returns the comment limit given by the user. */
+  private int getCommentLimit(HttpServletRequest request) {
+    String commentLimitString = request.getParameter("comment-limit");
+
+    int commentLimit;
+
+    try {
+      commentLimit = Integer.parseInt(commentLimitString);
+    } catch (NumberFormatException e) {
+      // Set default value for comment limit if parsing fails
+      return commentLimit = 10;
```

Comment:
Nit: Several of these values (0, 10, 100) could be top level constants. 
Typically these are formatted as something like
`const DEFAULT_COMMENT_LIMIT = 10;`